src/CodeStyle/Core/Analyzers/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,100 @@
+Microsoft.CodeAnalysis.Internal.Editing.DeclarationKind
```

Comment:
Have we confirmed with @sharwell that we want to consider this package shipped?

Reply:
I have not. @sharwell can you comment here if this should be reverted or not?

Reply:
I'm not entirely sure why these items are declared `public`. They appear to be in an internal namespace as an indication that they are not part of the package's official API. The design of the package is that it ships in stable forms for use, but it's not supported to reference the package APIs in other contexts.

Reply:
It looks like the `Internal` namespace portion is a lie. The type was marked public and left public for the CODESTYLE layer. Is that not intended?

Reply:
https://github.com/dotnet/roslyn/blob/release/dev17.4/src/Workspaces/Core/Portable/Editing/DeclarationKind.cs

Reply:
I'll just revert these to get this in. We can discuss this as a separate change if needed